lib/LedgerSMB/Upgrade_Tests.pm

```diff
@@ -106,14 +106,23 @@ Repair query table to run once per result.
 
 has table => (is => 'ro', isa => 'Str', required => 0);
 
-
 =item selectable_values
 
 Repair query column to get the values from
 
 =cut
 
-has selectable_values => (is => 'ro', isa => 'Str', required => 0);
+has selectable_values => (is => 'ro', isa => 'HashRef', required => 0);
+
+=item insert
+
+Allow insert instead of update. This to set defaults on a very limited subset
```

Comment:
Allow, to me, means that both will be accepted if the boolean is true, which contrasts with "instead of".

Can we be more specific? Is it either/or or is it update or insert+update?